wg-data/charter.md

```diff
@@ -0,0 +1,66 @@
+# WG Data Charter
+
+This charter adheres to the conventions, roles, and organisation management outlined in [wg-governance] for the Working Group "Data".
+
+## Scope
+
+The WG "Data" is focused on enhancing the support for Data/metadata-related tasks within Kubeflow, with a specific focus on the [Spark operator](https://github.com/kubeflow/community/pull/672) and [Model Registry](https://github.com/kubeflow/kubeflow/issues/7396).
+The group aims to streamline data processing workflows, facilitate efficient data lifecycle and ML models' metadata management, while ensuring seamless integration with other Kubeflow components.
+
+An additional goal of the group is to offer a common ground for data/metadata-related topics in the MLOps orbit that didn't have a more specific working group yet, so they can "incubate as one", coherent effort.
+
+For example: Data Preparation, Feature Store, and Model Registry have been recently discussed in the Kubeflow community while not mature enough yet to have their own working group, they can be nurtured together as part of this WG.
+
+### In scope
+
+#### Code, Binaries, and Services
+
+- Onboarding and maintenance of the Spark operator for scalable and distributed data processing.
+[See also](https://github.com/kubeflow/community/pull/672)
+- Continued development of the Model Registry to manage and version machine learning models efficiently.
+[See also](https://github.com/kubeflow/kubeflow/issues/7396)
```

Comment:
Let's add actual links to the repos here.
https://github.com/kubeflow/model-registry
https://github.com/kubeflow/spark-operator

Reply:
done with c8ef0c01779d18fc52dcc138f8417e3140af7ae3